src/utils/request/onUnhandledRequest.ts

```diff
@@ -11,10 +11,10 @@ export type OnUnhandledRequest =
 
 export function onUnhandledRequest(
   request: MockedRequest,
-  onUnhandledRequest: OnUnhandledRequest = 'bypass',
+  onUnhandledReq: OnUnhandledRequest = 'bypass',
```

Comment:
Let's name this parameter `handler`.

That way we don't have to shorten it while still preserving its semantic meaning: a function that handles an `onUnhandled` event.